static/app/views/alerts/rules/details/body.tsx

```diff
@@ -346,57 +336,55 @@ export default class DetailsBody extends React.Component<Props> {
+                      </ChartControls>
+                    </HeaderItem>
+                    {projects && projects.length && (
                       <HeaderItem>
-                        <Heading noMargin>{t('Display')}</Heading>
-                        <ChartControls>
-                          <DropdownControl
-                            label={
-                              getDynamicText({
-                                fixed: 'Oct 14, 2:56 PM — Oct 14, 4:55 PM',
-                                value: timePeriod.display,
-                              }) ?? '' // we should never get here because timePeriod.display is typed as always defined
-                            }
-                          >
-                            {TIME_OPTIONS.map(({label, value}) => (
-                              <DropdownItem
-                                key={value}
-                                eventKey={value}
-                                isActive={
-                                  !timePeriod.custom && timePeriod.period === value
-                                }
-                                onSelect={this.props.handleTimePeriodChange}
-                              >
-                                {label}
-                              </DropdownItem>
-                            ))}
-                          </DropdownControl>
-                        </ChartControls>
-                      </HeaderItem>
-                      {projects && projects.length && (
-                        <HeaderItem>
-                          <Heading noMargin>{t('Project')}</Heading>
+                        <Heading noMargin>{t('Project')}</Heading>
 
-                          <IdBadge avatarSize={16} project={projects[0]} />
-                        </HeaderItem>
-                      )}
-                      <HeaderItem>
-                        <Heading noMargin>
-                          {t('Time Interval')}
-                          <Tooltip
-                            title={t(
-                              'The time window over which the metric is evaluated.'
-                            )}
-                          >
-                            <IconInfo size="xs" color="gray200" />
-                          </Tooltip>
-                        </Heading>
-
-                        <RuleText>{this.getTimeWindow()}</RuleText>
+                        <IdBadge avatarSize={16} project={projects[0]} />
                       </HeaderItem>
-                    </HeaderGrid>
-                  </HeaderContainer>
+                    )}
+                    <HeaderItem>
+                      <Heading noMargin>{t('Environment')}</Heading>
+                      <RuleText>{rule.environment ?? 'All'}</RuleText>
+                    </HeaderItem>
+                    <HeaderItem>
+                      <Heading noMargin>
+                        {t('Time Interval')}
+                        <StyledTooltip
```

Comment:
Should we use the component https://github.com/getsentry/sentry/blob/master/static/app/components/questionTooltip.tsx here?

Reply:
Ohhh I didn’t know about this! Can do.